src/utils/cronJobs.ts

```diff
@@ -16,4 +19,19 @@ export function startCronJobs() {
       isFetching = false
     }
   })
+
+  /**
+   * check network health every hour
+   */
+  if (process.env.SLACK_WEBHOOK_SECRET_URL) {
```

Comment:
This check prevents the healthcheck cronJob from being created. The reason for this check was to keep the app running even without configuring healthcheck notifications at all (since this is not the main function).

Reply:
So we are looking for a feature flag here. I'd implement it as something like `FF_NOTIFY_HEALTH_SLACK`